testsuite/tests/tool-ocamldoc/Alert_toplevel2.mli

```diff
@@ -0,0 +1,5 @@
+(* TEST
+   * ocamldoc with html
+*)
+
+[@@@alert deprecated "foo"]
```

Comment:
I would propose to keep the same docstring comment that the `Alert_toplevel.mli` test in order to have exactly the same reference file.

Reply:
That's what I wanted to test actually, that the docstring doesn't interfere with the alert. That it's using a different kind of alert is a bonus. (it should be covered already in the other tests because it's using the same code)